core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -395,12 +394,18 @@ public synchronized void flush() throws IOException, JournalClosedException {
 
   @Override
   public synchronized void close() throws IOException {
```

Comment:
If `close` does not throw an exception anymore, it would be good to remove the `throws` from the method signature. This way callers of this method do not have to handle the exception.
```suggestion
  public synchronized void close() {
```

Reply:
right. updated this and all callers.